docs/APIOverview.mdx

```diff
@@ -3,15 +3,15 @@ id: api-overview
 title: API Overview
 ---
 
-This overview only shows you how to use Casbin APIs and doesn't explain how Casbin is installed and how it works. You can find those tutorials here: [installation of Casbin](https://casbin.org/docs/en/get-started) and [how Casbin works](https://casbin.org/docs/en/how-it-works). So, when you start reading this tutorial, we assume that you have fully installed and imported Casbin into your code.
+This overview only shows you how to use Casbin APIs and doesn't explain how Casbin is installed and how it works. You can find those tutorials here: [installation of Casbin](https://casbin.io/docs/get-started) and [how Casbin works](https://casbin.io/docs/how-it-works). So, when you start reading this tutorial, we assume that you have fully installed and imported Casbin into your code.
```

Comment:
Use relative path like `/docs/get-started` everywhere, so theoretically we will not see any `https://casbin.io/docs/xxx`. Plz replace all